sklearn/utils/testing.py

```diff
@@ -588,7 +588,8 @@ def is_abstract(c):
     path = sklearn.__path__
     for importer, modname, ispkg in pkgutil.walk_packages(
             path=path, prefix='sklearn.', onerror=lambda x: None):
-        if ".tests." in modname:
+        # Prevent adding *SearchCV estimators from the old grid_search module
+        if (".tests." in modname) or ("grid_search" in modname):
```

Comment:
shouldn't that mean we should try to fix the sorting, not skip the module? This seems to not cover the old module as much, right?